modules/game/src/main/Game.scala

```diff
@@ -116,7 +116,7 @@ case class Game(
    * It starts at 1, and is incremented after Black's move.
    * NOTE: Duplicates chess.Game.fullMoveNumber (avoids loading toChess)
    */
-  def fullMoveNumber: Int = 1 + turns / 2
+  def fullMoveNumber: Int = 1 + (turns - 1) / 2
```

Comment:
Yeah, this field *is* dangerous to change though: https://github.com/ornicar/scalachess/blob/691e6fa10f39f581f3f7e65b2b529ea172c4b3a8/src/main/scala/Game.scala#L54 this one is only for aesthetics as far as I can tell.